weave-js/src/components/Panel2/PanelPlot/PanelPlot.tsx

```diff
@@ -936,2851 +544,37 @@ const PanelPlotConfigInner: React.FC<PanelPlotProps> = props => {
-      isAssignableTo(dimTypes.y, maybe('number'))
-    ) {
-      mark = 'bar';
-    } else if (
-      isAssignableTo(dimTypes.x, maybe('number')) &&
-      isAssignableTo(dimTypes.y, union(['string', 'date']))
-    ) {
-      mark = 'bar';
-    } else if (
-      isAssignableTo(dimTypes.x, list(maybe('number'))) &&
-      isAssignableTo(dimTypes.y, union(['string', 'number']))
-    ) {
-      mark = 'boxplot';
-    } else if (
-      isAssignableTo(dimTypes.y, list(maybe('number'))) &&
-      isAssignableTo(dimTypes.x, union(['string', 'number']))
-    ) {
-      mark = 'boxplot';
-    } else if (
-      isAssignableTo(dimTypes.x, list('number')) &&
-      isAssignableTo(dimTypes.y, list('number'))
-    ) {
-      mark = 'line';
-    }
-  }
-
-  return mark;
-}
-
-type VegaTimeUnit = 'yearweek' | 'yearmonth';
-
-// Color axis type gets a separate function because color axes can share multiple scales - one for each axis type.
-function getColorAxisType(
-  series: SeriesConfig,
-  vegaReadyTable: TableState.TableState
-): PlotState.VegaAxisType | undefined {
-  let colorAxisType: PlotState.VegaAxisType | undefined;
-  const dimTypes = PlotState.getDimTypes(series.dims, vegaReadyTable);
-  if (dimTypes.color != null) {
-    if (isAssignableTo(dimTypes.color, oneOrMany(maybe('number')))) {
-      colorAxisType = 'quantitative';
-    } else if (
-      isAssignableTo(
-        dimTypes.color,
-        oneOrMany(maybe(union(['string', 'boolean'])))
-      )
-    ) {
-      colorAxisType = 'nominal';
-    }
-  }
-  return colorAxisType;
-}
-
-function getAxisTimeUnit(isDashboard: boolean): VegaTimeUnit {
-  return isDashboard ? 'yearweek' : 'yearmonth';
-}
-/*
-const useMergeTables = makePromiseUsable(
-  (
-    currentData: _.Dictionary<any>[][] | undefined,
-    newFlatPlotTables: _.Dictionary<any>[][],
-    isLoading: boolean
-  ): Promise<_.Dictionary<any>[][]> => {
-    if (!isLoading && currentData) {
-      return Promise.all(
-        newFlatPlotTables.map((table, i) =>
-          mergeRealizedTableData(currentData?.[i] ?? [], table)
-        )
-      );
-    }
-    return Promise.resolve(currentData || []);
-  }
-);
-
-const useLatestData = (
-  newFlatPlotTables: _.Dictionary<any>[][],
-  isLoading: boolean,
-  series: SeriesConfig[]
-) => {
-  const [latestData, setLatestData] = useState<_.Dictionary<any>[][]>([]);
-  const seriesRef = useRef<SeriesConfig[]>(series);
-  const {result: mergedTables, loading} = useMergeTables(
-    latestData,
-    newFlatPlotTables,
-    isLoading
-  );
-
-  if (seriesRef.current !== series) {
-    // invalidate prior data
-    seriesRef.current = series;
-    setLatestData([]);
-    return {latestData: [], loading: true};
-  } else if (mergedTables !== latestData && !loading) {
-    setLatestData(mergedTables);
-  } else if (
-    loading &&
-    latestData.length === 0 &&
-    newFlatPlotTables.length > 0
-  ) {
-    setLatestData(newFlatPlotTables);
-  }
-
-  return {latestData, loading};
-};
-*/
```

Comment:
We should keep this in a commented out form in case we want to re-enable it later on